ModrinthUpdateChecker fails when not a single version is "release"

**Describe the bug**
Currently for 1.21 there is only a beta release and when checking versions it filters versions and tries to .get(0) in a list of 0.

```log
[22:11:50 WARN]: [NoxesiumUtils] Failed to fetch version from Modrinth.
[22:11:50 WARN]: java.util.concurrent.CompletionException: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
[22:11:50 WARN]: 	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
[22:11:50 WARN]: 	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
[22:11:50 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:722)
[22:11:50 WARN]: 	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
[22:11:50 WARN]: 	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2179)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//kong.unirest.apache.ApacheAsyncClient$1.completed(ApacheAsyncClient.java:86)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//kong.unirest.apache.ApacheAsyncClient$1.completed(ApacheAsyncClient.java:79)
[22:11:50 WARN]: 	at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:122)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseCompleted(DefaultClientExchangeHandlerImpl.java:182)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.nio.protocol.HttpAsyncRequestExecutor.processResponse(HttpAsyncRequestExecutor.java:448)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.nio.protocol.HttpAsyncRequestExecutor.inputReady(HttpAsyncRequestExecutor.java:338)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:265)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:87)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:40)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:121)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591)
[22:11:50 WARN]: 	at java.base/java.lang.Thread.run(Thread.java:1583)
[22:11:50 WARN]: Caused by: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
[22:11:50 WARN]: 	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
[22:11:50 WARN]: 	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
[22:11:50 WARN]: 	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
[22:11:50 WARN]: 	at java.base/java.util.Objects.checkIndex(Objects.java:385)
[22:11:50 WARN]: 	at java.base/java.util.ArrayList.get(ArrayList.java:427)
[22:11:50 WARN]: 	at NoxesiumUtils-2.0.0.jar//me.superneon4ik.noxesiumutils.modules.ModrinthUpdateChecker.lambda$checkForUpdates$1(ModrinthUpdateChecker.java:77)
[22:11:50 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[22:11:50 WARN]: 	... 19 more
```

--- a/src/main/java/me/superneon4ik/noxesiumutils/modules/ModrinthUpdateChecker.java
+++ b/src/main/java/me/superneon4ik/noxesiumutils/modules/ModrinthUpdateChecker.java
@@ -55,6 +55,7 @@ public class ModrinthUpdateChecker {
         }.runTaskTimerAsynchronously(NoxesiumUtils.getPlugin(), 1, refreshTicks);
     }
 
+    @SuppressWarnings("deprecation")
     public CompletableFuture<VersionStatus> checkForUpdates() {
         CompletableFuture<VersionStatus> future = new CompletableFuture<>();
         String uri = "https://api.modrinth.com/v2/project/%s/version?game_versions=%%5B%%22%s%%22%%5D"
@@ -64,17 +65,18 @@ public class ModrinthUpdateChecker {
             Gson gson = new Gson();
             var data = response.getBody().getArray().toString();
             var availableVersionsArray = gson.fromJson(data, ModrinthVersion[].class);
-            if (availableVersionsArray.length == 0) {
+
+            var availableVersions = new ArrayList<>(Arrays.stream(availableVersionsArray)
+                    .filter(v -> v.version_type.equalsIgnoreCase("release")).toList());
+            availableVersions.sort(ModrinthVersion::compareDatePublishedTo);
+            
+            if (availableVersions.isEmpty()) {
                 latestStatus = VersionStatus.NOT_FOUND;
                 future.complete(VersionStatus.NOT_FOUND);
                 return;
             }
 
-            var availableVersions = new ArrayList<>(Arrays.stream(availableVersionsArray)
-                    .filter(v -> v.version_type.equalsIgnoreCase("release")).toList());
-            availableVersions.sort(ModrinthVersion::compareDatePublishedTo);
-
-            if (availableVersions.get(0).version_number.toLowerCase().endsWith(NoxesiumUtils.getPlugin().getDescription().getVersion().toLowerCase())) {
+            if (availableVersions.getFirst().version_number.toLowerCase().endsWith(NoxesiumUtils.getPlugin().getDescription().getVersion().toLowerCase())) {
                 latestStatus = VersionStatus.LATEST;
                 future.complete(VersionStatus.LATEST);
                 return;
@@ -89,7 +91,7 @@ public class ModrinthUpdateChecker {
 
             future.complete(VersionStatus.OUTDATED);
             latestStatus = VersionStatus.OUTDATED;
-            latestKnownVersion = availableVersions.get(0).version_number;
+            latestKnownVersion = availableVersions.getFirst().version_number;
         }).exceptionally(throwable -> {
             latestStatus = VersionStatus.ERROR;
             future.complete(VersionStatus.ERROR);
@@ -104,6 +106,7 @@ public class ModrinthUpdateChecker {
         sendVersionMessage(sender, latestStatus);
     }
 
+    @SuppressWarnings("deprecation")
     public void sendVersionMessage(CommandSender sender, VersionStatus versionStatus) {
         if (latestKnownVersion == null) latestKnownVersion = NoxesiumUtils.getPlugin().getDescription().getVersion();
         sender.sendMessage(Component.text("Running NoxesiumUtils v" + NoxesiumUtils.getPlugin().getDescription().getVersion(), NamedTextColor.AQUA));

